conda/plugins/virtual_packages/cuda.py

```diff
@@ -35,13 +91,15 @@ def cuda_version():
             "libcuda.so",  # check library path first
             "/usr/lib64/nvidia/libcuda.so",  # RHEL/Centos/Fedora
             "/usr/lib/x86_64-linux-gnu/libcuda.so",  # Ubuntu
+            "/usr/lib/wsl/lib/libcuda.so",  # WSL
```

Comment:
The fix for WSL search path was added in #11626, but was accidentally removed in #11854. Readd it back here.

- #11626
- #11854

Reply:
OH wow, thanks for catching that.